test/transforms/test_index_to_mask.py

```diff
@@ -0,0 +1,32 @@
+import pytest
```

Comment:
merge into single file `test_mask.py`.

Reply:
so I tried this initially but had a build failure mentioning that there was a conflict with another `test_mask.py` in `test/utils` 

Let me dig up the log from that run in case it makes more sense to you.

Reply:
We can name it like `test_mask_transform.py`.